docs/registry/access-mgmt-registry/README.adoc

```diff
@@ -137,11 +139,39 @@ You can edit or remove the user roles assigned in your {registry} instances that
 .Procedure
 . In the web console, go to *{registry}* > *{registry} Instances* and click the name of the {registry} instance that you want to remove a user role for.
 . Click the *Access* tab to view the accounts and roles assigned for this instance.
-. Select the options menu (three vertical dots) next to the assigned *Role* name: 
-.. To change to a different role, click *Edit* and select the new user role, for example, *Viewer* for read-only access to this instance.  
+. Select the options menu (three vertical dots) next to the assigned *Role* name:
+.. To change to a different role, click *Edit* and select the new user role, for example, *Viewer* for read-only access to this instance.
 .. To remove the currently assigned role, click *Remove* and confirm in the dialog.
 
+[id="proc-user-account-access_{context}"]
+== Using RBAC to allow users to view other user accounts
```

Comment:
Suggest changing heading to "Allowing users to view other user accounts" for consistency with the other headings in this guide.

All of the sections in this doc are using SR RBAC (as opposed to HCC RBAC) and don't mention RBAC it in the heading.

Reply:
Thanks @smccarthy-ie. 
For the related Kafka procedure, @sterobin thought we should have _RBAC_ in the title. It would be good to be consistent. WDYT, Stetson? Use _Allowing users to view other user accounts_ for both?

Reply:
Sorry I missed this @PaulRMellor . I guess that's a good point that @smccarthy-ie makes. So okay to drop the RBAC for consistency.

Reply:
Thanks @sterobin -- Updated as @smccarthy-ie  suggested